pykube/http.py

```diff
@@ -32,13 +32,15 @@ def build_session(self):
         s = requests.Session()
         if "certificate-authority" in self.config.cluster:
             s.verify = self.config.cluster["certificate-authority"].filename()
+
         if "token" in self.config.user and self.config.user["token"]:
             s.headers["Authorization"] = "Bearer {}".format(self.config.user["token"])
-        else:
+        elif "client-certificate" in self.config.user:
```

Comment:
Let's check for both `client-certificate` and `client-key` before executing this branch of code.